contracts/HubPool.sol

```diff
@@ -71,9 +71,6 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     // can be submitted.
     RootBundle public rootBundleProposal;
 
-    // Whether the bundle proposal process is paused.
```

Comment:
Any changes in the order of state variables is done to take advantage of variable packing